src/OmniSharp.MSBuild/ProjectManager.cs

```diff
@@ -263,7 +279,13 @@ private void AddProject(ProjectFileInfo projectFileInfo)
 
             _projectFiles.Add(projectFileInfo);
 
-            var projectInfo = projectFileInfo.CreateProjectInfo();
+            var projectInfo = projectFileInfo.CreateProjectInfo(_assemblyLoader);
```

Comment:
ah.. this is actually the one that create project? sorry. that sure it should have assemblyLoader.